src/sentry/integrations/vercel/client.py

```diff
@@ -11,6 +11,10 @@ class VercelClient(ApiClient):
     TEAMS_URL = "/v1/teams/%s"
     USER_URL = "/www/user"
     PROJECTS_URL = "/v4/projects/"
+    ENV_VAR_URL = "/v4/projects/%s/env"
+    GET_ENV_VAR_URL = "/v5/projects/%s/env"
```

Comment:
@ceorourke It's so weird to me that they have different versioning for `GET` and `POST`. I'm gonna ask about it...